subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -100,6 +100,47 @@ intended for internal use and was never intended to be used as part of the publi
 
 Starting in Gradle 9.0, this method will be removed without replacement.
 
+[[test_task_default_classpath]]
+==== Relying on Test task default testClassesDirs and classpath
+
+By default, when applying the `JavaPlugin`, standalone `Test` tasks are configured such that their `testClassesDirs`
```

Comment:
Instead of using the plugin's class name, we should use it's plugin id and link to its user manual page